etc/config/sway.amazon.properties

```diff
@@ -10,10 +10,19 @@ group.sway.irArg=--ir final
 group.sway.supportsBinary=true
 group.sway.supportsAsm=true
 group.sway.asmArg=--asm all
+group.sway.options=--offline
 
 compiler.swayv0667.exe=/opt/compiler-explorer/sway-0.66.7/forc-binaries/forc
 compiler.swayv0667.semver=0.66.7
 compiler.swayv0667.name=sway 0.66.7
 
 # Basic tools that might be useful
 tools=
+
+# Add the standard library
```

Comment:
Why have this here? Must the user pick this themselves? these libraries are the ones a user could choose themselves. Would a user ever pick the "wrong" version? or choose not to pi8ck it? you're hardcoding this anyway? 

I'd suggest removing this.

Reply:
Good point, thanks for clarifying this. Would get very confusing needing to select this as a library.